### ISSUE TITLE
exceptions.AttributeError: 'NetTestCaseWithLocalOptions' object has no attribute '_start_time'

### ISSUE THREAD

**Issue by darkk** [TheTorProject/ooni-probe#626]
I introduced non-fatal bug in 15bfb0312030a79c1985564dcbf7ee18101388b4 while solving #612 that is complaining about bad input twice that looks confusing:

```
2016-10-12 03:00:02,590799+0200 [DNSDatagramProtocol (UDP)] Starting this task <generator object generateMeasurements at 0x7fe85cfcfd70>
2016-10-12 03:00:02,591635+0200 [DNSDatagramProtocol (UDP)] Finished test setup
2016-10-12 03:00:02,592137+0200 [DNSDatagramProtocol (UDP)] Unhandled Error
»       Traceback (most recent call last):
»         File "/usr/local/lib/python2.7/dist-packages/twisted/internet/defer.py", line 306, in addCallbacks
»           self._runCallbacks()
»         File "/usr/local/lib/python2.7/dist-packages/twisted/internet/defer.py", line 587, in _runCallbacks
»           current.result = callback(current.result, *args, **kw)
»         File "/usr/local/lib/python2.7/dist-packages/ooniprobe-2.0.0rc4.dev0-py2.7.egg/ooni/managers.py", line 61, in <lambda>
»           d.addCallback(lambda _: self._scheduleNextTask())
»         File "/usr/local/lib/python2.7/dist-packages/ooniprobe-2.0.0rc4.dev0-py2.7.egg/ooni/managers.py", line 66, in _scheduleNextTask
»           task = next(self._tasks)
»       --- <exception caught here> ---
»         File "/usr/local/lib/python2.7/dist-packages/ooniprobe-2.0.0rc4.dev0-py2.7.egg/ooni/nettest.py", line 601, in generateMeasurements
»           test_input)
»         File "/usr/local/lib/python2.7/dist-packages/ooniprobe-2.0.0rc4.dev0-py2.7.egg/ooni/nettest.py", line 554, in makeMeasurement
»           measurement = Measurement(test_instance, test_method, test_input)
»         File "/usr/local/lib/python2.7/dist-packages/ooniprobe-2.0.0rc4.dev0-py2.7.egg/ooni/tasks.py", line 114, in __init__
»           self.testInstance.setUp()
»         File "/usr/local/lib/python2.7/dist-packages/ooniprobe-2.0.0rc4.dev0-py2.7.egg/ooni/nettests/blocking/web_connectivity.py", line 181, in setUp
»           raise AbsentHostname('No hostname', self.input)
»       ooni.nettests.blocking.web_connectivity.AbsentHostname: ('No hostname', 'http:///www.blackblocg.info')
»       
2016-10-12 03:00:02,592700+0200 [DNSDatagramProtocol (UDP)] [!] Failed to run <<class 'ooni.nettest.NetTestCaseWithLocalOptions'> inputs=<generator object inputProcessor at 0x7fe85cf869b0>> test_web_connectivity http:///www.blackblocg.info
2016-10-12 03:00:02,593463+0200 [DNSDatagramProtocol (UDP)] Finished test setup
2016-10-12 03:00:02,594047+0200 [DNSDatagramProtocol (UDP)] Running <<class 'ooni.nettest.NetTestCaseWithLocalOptions'> inputs=<generator object inputProcessor at 0x7fe85cf869b0>> test_web_connectivity
2016-10-12 03:00:02,594438+0200 [DNSDatagramProtocol (UDP)]·
2016-10-12 03:00:02,594721+0200 [DNSDatagramProtocol (UDP)] Starting test for http://2ch.hk
2016-10-12 03:00:02,594961+0200 [DNSDatagramProtocol (UDP)] * doing DNS query for 2ch.hk
2016-10-12 03:00:02,596298+0200 [-] Unhandled error in Deferred:
2016-10-12 03:00:02,596697+0200 [-] Unhandled Error
»       Traceback (most recent call last):
»         File "/usr/local/lib/python2.7/dist-packages/ooniprobe-2.0.0rc4.dev0-py2.7.egg/ooni/managers.py", line 66, in _scheduleNextTask
»           task = next(self._tasks)
»         File "/usr/local/lib/python2.7/dist-packages/ooniprobe-2.0.0rc4.dev0-py2.7.egg/ooni/nettest.py", line 616, in generateMeasurements
»           @post.addBoth
»         File "/usr/local/lib/python2.7/dist-packages/twisted/internet/defer.py", line 340, in addBoth
»           callbackKeywords=kw, errbackKeywords=kw)
»         File "/usr/local/lib/python2.7/dist-packages/twisted/internet/defer.py", line 306, in addCallbacks
»           self._runCallbacks()
»       --- <exception caught here> ---
»         File "/usr/local/lib/python2.7/dist-packages/twisted/internet/defer.py", line 587, in _runCallbacks
»           current.result = callback(current.result, *args, **kw)
»         File "/usr/local/lib/python2.7/dist-packages/ooniprobe-2.0.0rc4.dev0-py2.7.egg/ooni/nettest.py", line 618, in set_runtime
»           runtime = time.time() - test_instance._start_time
»       exceptions.AttributeError: 'NetTestCaseWithLocalOptions' object has no attribute '_start_time'
»       
2016-10-12 03:00:02,597345+0200 [DNSDatagramProtocol (UDP)] Finished test setup
2016-10-12 03:00:02,597921+0200 [DNSDatagramProtocol (UDP)] Running <<class 'ooni.nettest.NetTestCaseWithLocalOptions'> inputs=<generator object inputProcessor at 0x7fe85cf869b0>> test_web_connectivity
2016-10-12 03:00:02,598277+0200 [DNSDatagramProtocol (UDP)]·
2016-10-12 03:00:02,598533+0200 [DNSDatagramProtocol (UDP)] Starting test for http://3a-pycb.livejournal.com
```

**Comment by hellais** [comment]
Beautiful, am able to stably reproduce this issue with:

```
PYTHONPATH=`pwd` python ooni/scripts/ooniprobe.py web_connectivity --url http:///www.blackblocg.info
```

**Comment by darkk** [comment]
Should be fixed by #629
